modules/test/playwright/tests/object-web/listTypeDefinitions.spec.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {listTypeDefinitionsPagesTest} from '../../fixtures/listTypeDefinitionsPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {siteSettingsPageTests} from '../../fixtures/siteSettingsPagesTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	listTypeDefinitionsPagesTest,
+	loginTest(),
+	siteSettingsPageTests
+);
+
+test.describe('manage picklists inside the picklists portlet', () => {
+	test('can create a picklist', async ({
+		apiHelpers,
+		listTypeDefinitionPage,
+		page,
+	}) => {
+		await page.goto('/');
+
+		await listTypeDefinitionPage.goto();
+
+		const listTypeDefinitionName = 'picklist' + getRandomInt();
+
+		const listTypeDefinition = await listTypeDefinitionPage.createPicklist(
+			listTypeDefinitionName
+		);
+
+		await expect(page.getByText(listTypeDefinition.name)).toBeVisible();
+
+		// Clean up
```

Comment:
We are moving to a more resilient clean up. See example [here](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/object-web/objectValidations.spec.ts#L23-L46). Store created entities in global variables and use inside `afterEach`.